### PR TITLE
Add Google Analytics tracking snippet

### PR DESCRIPTION
**Pull request by smoothcontract**
### Context
https://dfedigital.atlassian.net/browse/GET-352

### Changes proposed in this pull request
Refactored the config for smart survey so that `ENV` no longer needs to be stubbed - this was conflicting when I added the GA snippet as there are then two ENV vars in the application layout. So now `Rails.configuration` is updated with the two relevant environment variables which makes it easier to stub in tests.

Moved logic for including the relevant markup to the partial itself which makes the main application layout a bit cleaner (along with other minor cleanup).

### Guidance to review
The new env var GOOGLE_ANALYTICS_TRACKING_ID will only be defined on the QA environment, but I'll temporarily set that up on the Heroku review app once deployed so it can be tested. I'm not sure if this is enough to start sending tracking info to GA - suspect we'll need to do some more steps to trust the tracking source, particularly if it comes from heroku review app.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -31,5 +31,8 @@ class Application < Rails::Application
     config.exceptions_app = routes
 
     config.i18n.default_locale = :'en-GB'
+
+    config.google_analytics_tracking_id = ENV['GOOGLE_ANALYTICS_TRACKING_ID']
+    config.smart_survey_user_feedback_link = ENV['USER_FEEDBACK_SMART_SURVEY_LINK']
```
Comment (tavy88) [inline config/application.rb]: Should we do the same with our `NotifyService` ?

**Comment by tavy88** [comment]
Also rubocop doesn't look happy.

File: config/application.rb (hunk 1 above)
Comment (smoothcontract) [reply config/application.rb]: Good idea - just added that too

File: .env.test [hunk 2]
```diff
@@ -1,2 +1,3 @@
 USER_FEEDBACK_SMART_SURVEY_LINK=https://www.smartsurvey.co.uk/s/8IBQ1/
 NOTIFY_API_KEY=test
```
Comment (tavy88) [inline .env.test]: I don't think we need to give the `test` value here. I know it's not your change, but I would like if we could remove it.

Comment (smoothcontract) [reply .env.test]: Outside the scope of this PR I think, but we should discuss again.

**Pull request merged**